lighthouse-core/runner.js

```diff
@@ -177,6 +177,18 @@ class Runner {
             debugString: `Required ${artifactName} gatherer did not run.`
           });
         }
+
+        // If artifact was an error, it must be non-fatal (or gatherRunner would
+        // have thrown). Output error result on behalf of audit.
+        if (artifacts[artifactName] instanceof Error) {
```

Comment:
Any of the same machinery for audits? Or are we still treating all errors there as fatal?